### PR TITLE
Create fwlink URLs for properties with HelpUrls.

### LINKED ISSUE #7262: Create fwlink URLs for all property HelpUrls (1)
The properties shown in the new Project Properties UI and Launch Profile UI are specified in XAML rule tiles. Each property may have a `HelpUrl` associated with it. These URLs should go via the fwlink redirection service, so we can retarget them in future if they break due to changes in the location of documentation.

### PR DESCRIPTION
**Pull request by melytc**
Fixes #7262.

I went through our rule files to 
1. Create a fwlink URL if needed.
2. Verify that the URL doesn't reference a particular cutlure (i.e. `en-us`) in our link manager website.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/project-system/pull/7324)

### CODE REVIEW

**Review by drewnoakes — APPROVED** [review]

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml [hunk 1]
```diff
@@ -342,11 +335,10 @@
     </StringProperty.ValueEditors>
   </StringProperty>
 
-  <!-- TODO create fwlink -->
   <StringProperty Name="PostBuildEvent"
                   DisplayName="Post-build event"
                   Description="Specifies commands that run after the build completes. Does not run if the build failed. Use 'call' to invoke .bat files. A non-zero exit code will fail the build."
-                  HelpUrl="https://docs.microsoft.com/visualstudio/ide/how-to-specify-build-events-csharp?view=vs-2019"
+                  HelpUrl="https://go.microsoft.com/fwlink/?linkid=2165773"
```
[the remark this answers is not in the document]
Comment (melytc) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml:341]: Just changed it in our link manager website (it's nice the resulting fwlink stays the same 😄). Thanks!

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml [hunk 2]
```diff
@@ -172,11 +171,10 @@
     </BoolProperty.Metadata>
   </BoolProperty>
 
-  <!-- TODO use fwlink for HelpUrl -->
   <BoolProperty Name="UseWPF"
                 DisplayName="Windows Presentation Foundation"
                 Description="Enable WPF for this project."
-                HelpUrl="https://docs.microsoft.com/dotnet/core/project-sdk/msbuild-props-desktop"
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2165974"
```
[the remark this answers is not in the document]
Comment (melytc) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml:177]: I could point at the specific section on that page ([UseWPF](https://go.microsoft.com/fwlink/?linkid=2165974), [UseWindowsForms](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props-desktop#usewindowsforms)). They both live in the same page under `Microsoft.NET.Sdk.Desktop`.

[the remark this answers is not in the document]
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml:177]: Even if we have two fwlinks that point to the same place today, we may wish to make them point to different places in future.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml:341]: Perhaps you can show the link manager in the team meeting this week.

**Review by swesonga — APPROVED** [review]

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (melytc) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml]: Makes sense, added on the last commit.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml [hunk 3]
```diff
@@ -304,9 +302,9 @@
     </StringProperty.Metadata>
   </StringProperty>
 
-  <!-- TODO Documentation issue for this property: https://github.com/MicrosoftDocs/visualstudio-docs/issues/6643 -->
   <BoolProperty Name="IsPublishable"
                 DisplayName="Publishing"
                 Description="Allow publishing outside of Visual Studio, such as via &quot;dotnet publish&quot;."
+                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2165975"
```
Comment (MiYanni) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml:308]: Thank you! I had this on my radar. 😊

**Review by MiYanni — APPROVED** [review]

**Pull request merged**